actions.js

```diff
@@ -937,8 +983,8 @@ ActionManager.prototype.onAddBlock = function(block, ownerId, x, y) {
         firstBlock.changed();
         owner.scripts.adjustBounds();
     } else {
-        var def = this._customBlocks[ownerId],
-            editor = this._getCustomBlockEditor(ownerId),
+        var def = this._customBlocks[ownerId],  // ownerId?!?!?
```

Comment:
I have to check this out... However, it isn't related to this issue so it isn't necessarily blocking this PR